static/js/containers/DashboardPage.js

```diff
@@ -62,45 +58,42 @@ import {
   updateDocumentSentDate,
 } from '../actions/documents';
 import {
+  skipFinancialAid,
   startCalculatorEdit,
   updateCalculatorEdit,
 } from '../actions/financial_aid';
 import { setTimeoutActive } from '../actions/order_receipt';
 import { attachCoupon, setRecentlyAttachedCoupon } from '../actions/coupons';
 import {
-  startEmailEdit,
-  updateEmailEdit,
-  clearEmailEdit,
-  updateEmailValidation,
-  sendCourseTeamMail
-} from '../actions/email';
-import { emailValidation } from '../lib/validation/profile';
-import EmailCompositionDialog from '../components/email/EmailCompositionDialog';
+  getPearsonSSODigest,
+  pearsonSSOInProgress,
+  pearsonSSOFailure,
+  setPearsonError
+} from '../actions/pearson';
+import { calculatePrices } from '../lib/coupon';
+import { findCourseRun } from '../util/util';
 import { singleBtnDialogActions } from '../components/inputs/util';
+import { currencyForCountry } from '../lib/currency';
+import DocsInstructionsDialog from '../components/DocsInstructionsDialog';
+import CouponNotificationDialog from '../components/CouponNotificationDialog';
+import { generateSSOForm } from '../lib/pearson';
+import { COURSE_EMAIL_TYPE } from '../components/email/constants';
+import { COURSE_TEAM_EMAIL_CONFIG } from '../components/email/lib';
+import { withEmailDialog } from '../components/email/hoc';
+
 import type { UIState } from '../reducers/ui';
 import type { OrderReceiptState } from '../reducers/order_receipt';
 import type { DocumentsState } from '../reducers/documents';
-import type { CoursePricesState, DashboardState } from '../flow/dashboardTypes';
-import type {
-  AvailableProgram, AvailableProgramsState, CourseEnrollmentsState
-} from '../flow/enrollmentTypes';
 import type { FinancialAidState } from '../reducers/financial_aid';
 import type { CouponsState } from '../reducers/coupons';
-import type { EmailState } from '../flow/emailTypes';
+import type { PearsonAPIState } from '../reducers/pearson';
+import type { AllEmailsState } from '../flow/emailTypes';
 import type { ProfileGetResult } from '../flow/profileTypes';
 import type { Course, CourseRun } from '../flow/programTypes';
-import { skipFinancialAid } from '../actions/financial_aid';
-import { currencyForCountry } from '../lib/currency';
-import DocsInstructionsDialog from '../components/DocsInstructionsDialog';
-import CouponNotificationDialog from '../components/CouponNotificationDialog';
-import {
-  getPearsonSSODigest,
-  pearsonSSOInProgress,
-  pearsonSSOFailure,
-  setPearsonError
-} from '../actions/pearson';
-import { generateSSOForm } from '../lib/pearson';
-import type { PearsonAPIState } from '../reducers/pearson';
+import type { CoursePricesState, DashboardState } from '../flow/dashboardTypes';
+import type {
+  AvailableProgram, AvailableProgramsState, CourseEnrollmentsState
+} from '../flow/enrollmentTypes';
```

Comment:
if possible (and like not too much work) could you revert the changes to imports here to just reflect what's changed?